.travis.yml

```diff
@@ -40,6 +41,37 @@ stages:
 
 jobs:
   include:
+  # ppc64le: "recent" systems with ICB
+  #   note: when you PR, docker-cp provides, in the container, the branch associated with the PR (not master where there's nothing new)
+  #         1. docker create --name mobydick IMAGE CMD        <=> create a container (= instance of image) but container is NOT yet started
+  #         2. docker cp -a ${TRAVIS_BUILD_DIR} mobydick:/tmp <=> copy git repository (CI worker, checkout-ed on PR branch) into the container
+  #                                                               note: docker-cp works only if copy from/to containers (not images)
+  #         3. docker start -a mobydick                       <=> start to run the container (initialized with docker-cp)
+  - stage: ppc64le
+    dist: bionic
+    script: |
+      sudo docker pull ppc64le/debian                                                                                   \
```

Comment:
Please move all of that it to a script which can take arguments just like it is done for redhat

Also, not sure it should be first in the list :)